api_tests/waffle/views/test_waffle_list.py

```diff
@@ -43,7 +45,7 @@ def flag_url(self, url):
     def test_waffle_flag_no_filter(self, app, user, url, inactive_flag, active_flag):
         res = app.get(url, auth=user.auth)
         assert res.status_code == 200
-        assert len(res.json['data']) == 2
+        assert len(res.json['data']) == Flag.objects.all().count()
```

Comment:
I think this has to be done because extra flags are added via migration, but I'm not 100%. Regardless this is the behavior I would expect for this test.